src/controller/nwnx.ini

```diff
@@ -1,5 +1,5 @@
 # NWNX4 configuration file
-# These are the default values for NWNX4. 
+# These are the default values for NWNX4.
```

Comment:
Looks to be a spacing fix. We should setup a PR to fix and cleanup the look of this file at a point. There's spelling and  grammatical errors that might help with readability.

Reply:
We need to setup clang-format on the project, and have it automatically reformat the code (and check it's formatted with the CI)

However reformatting the code wil cause merge issues (or at least some overhead) if there's unmerged branches, so I'd better do it when the project isn't very active